go.mod

```diff
@@ -1,15 +1,14 @@
 module k8s.io/kube-openapi
 
-go 1.20
+go 1.21
```

Comment:
@liggitt are we able to safely bump this? Per https://github.com/kubernetes/kube-openapi/issues/463 I'd imagine we wanted to keep 1.20 and directly skip to 1.23 once that's the lowest supported k/k version.

Reply:
Edit: https://github.com/kubernetes/kube-openapi/pull/517#discussion_r1859757625, looks like the oldest go directive is k/k v1.29 @ go1.21